src/modules/site-v2/templates/tools/pairwise_indel_finder/submit.html

```diff
@@ -549,7 +549,7 @@ <h1 class="modal-title fs-5" id="generate-primers-modal-label">Thank You</h1>
   }
 
   // Submit the job, opening results in a new tab if applicable
-  submit_job(data, '#generate-primers-modal', true)
+  submit_job(data, '#generate-primers-modal', true, false)
```

Comment:
consider using a config object instead of sequencial booleans, (...,..., { foo: true, bar: false}) for clarity of order.

Reply:
Fixed in [27fe543](https://github.com/AndersenLab/CAENDR/pull/402/commits/27fe543f3e581ff847e79c2b5470b48b31e2d903)